include/lsst/gauss2d/evaluate.h

```diff
@@ -957,6 +958,8 @@ class GaussianEvaluator : public Object {
     }
 
 private:
+    // These are private because they originally return static objects that
+    // were (and are) not meant to be mutated, although they probably can be
     Data& IMAGE_NULL() const;
```

Comment:
Why not make the return value `const`?
Because "they probably can be" (mutated)
```suggestion
    Data const& IMAGE_NULL() const;
```

Reply:
I already have `IMAGE_NULL_CONST` for that. This is for internal use (hence the private).

Basically the only reason this exists is that I need to pass a non-const Image ref to some functions. For template args where this isn't actually needed I just don't mutate it; otherwise it is retrieved from an ImageArray with the at function.

I suppose I could change the internal functions to take a raw pointer and have it nullptr instead of a dummy Image at no real loss of performance or convenience.

Reply:
Actually it wasn't so difficult to change existing usage to pointer only (shared, not raw) so I did that and removed the method entirely. Being a private function, I think that doesn't warrant a minor version increment.